stages/cloud-provider/aws/sharedsvc/variables.tf

```diff
@@ -75,3 +75,9 @@ variable "cluster_domain" {
 variable "internal_cluster_domain" {
   default = "internal.services.liatr.io"
 }
+
+variable "github_runner_service_accounts" {
+  description = "a list of service accounts to apply IRSA trust policy to. each service account should be in the format `namespace:name`"
```

Comment:
If these require some special formatting, why not use a list of objects? Just curious, it's probably easier just to pass them through if this is the way that the github runner module consumes them.

Reply:
I'm not entirely sure. I think `namespace:name` is the format that the iam role expects; I suppose it was easier to specify a string to include than to pass a map that gets formatted in the final resource.